### PR TITLE
Improve error when missing sigs and atts

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-1689

Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
I should've broken up this PR into two... or at least two separate commits. Let me know if this is too hard to review as is, and I can break it up.

File: features/validate_image.feature [hunk 1]
```diff
@@ -81,10 +81,8 @@ Feature: evaluate enterprise contract
           "name": "Unnamed",
           "containerImage": "localhost:(\\d+)/acceptance/invalid-image-signature",
           "violations": [
-            {"msg": "no matching signatures:\ninvalid signature when validating ASN.1 encoded signature"},
-            {"msg": "no matching attestations:\nAccepted signatures do not match threshold, Found: 0, Expected 1"},
-            {"msg": "EV001: No attestation data, at github.com/hacbs-contract/ec-cli/internal/evaluation_target/application_snapshot_image/application_snapshot_image.go:47"},
-            {"msg": "no attestations available"}
+            {"msg": "No images signatures found matching the given public key. Verify the correct public key was provided, and a signature was created."},
```
Comment (simonbaird) [inline features/validate_image.feature:84]: Is it supposed to be `image signatures` or `images' signatures` maybe, with a possessive apostrophe?

**Review by simonbaird — APPROVED** [review]
Lgtm, but I don't know this code very well, so maybe seek a second opinion.

File: features/validate_image.feature (hunk 1 above)
Comment (lcarva) [reply features/validate_image.feature:84]: I think image signatures is the terminology I've seen used elsewhere.

**Review by zregvart — APPROVED** [review]

**Pull request merged**